src/pages/dashboard/promo/miniBanner/detail.js

```diff
@@ -15,19 +15,27 @@ import { DetailInfo, DetailListingProduct } from '../components';
 
 export function DetailMiniBanner() {
   const [detailInfo, setDetailInfo] = useState({});
+  const [isModalVisible, setIsModalVisible] = useState(false);
   const { id } = useParams();
+  const [form] = Form.useForm();
 
-  const { Products: products = [], status } = detailInfo;
+  const showModal = () => setIsModalVisible(true);
+  const closeModal = () => setIsModalVisible(false);
+
+  const { Products: products = [], status_admin: status } = detailInfo;
 
   const fetchDetailMiniBanner = async () => {
     try {
-      const {
+      let {
         data: { data: bannerData = {} },
       } = await MiniBannerService.getDetailMiniBanner(id);
       const { supplier_id } = bannerData;
       const {
         data: { data: { Supplier: supplier = {} } = {} },
       } = await SupplierService.getDetailSupplier(supplier_id);
+      bannerData.Products.forEach((val, idx) => {
+        bannerData.Products[idx].brand_name = supplier.brand_name;
+      });
```

Comment:
atau hasil mapping ini yg dibuat var baru, terus setDetailInfo nya pake var baru
```suggestion
      const bannerDataWithBrandName = bannerData.Products.forEach((val, idx) => {
        bannerData.Products[idx].brand_name = supplier.brand_name;
      });
```

Reply:
Baik akan direfactor Mba